### PR TITLE
tomcat 7.0.61

### PR DESCRIPTION
**Pull request by asaph**
I also changed all apache urls from http to https and switched from sha1 hashes to sha256.

### CODE REVIEW

File: tomcat7.rb [hunk 1]
```diff
@@ -1,16 +1,16 @@
 require 'formula'
 
 class Tomcat7 < Formula
-  homepage "http://tomcat.apache.org/"
-  url "http://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-7/v7.0.59/bin/apache-tomcat-7.0.59.tar.gz"
-  sha1 "c9f8d87a212091c33027f122aba90b78eb1c0a01"
+  homepage "https://tomcat.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=tomcat/tomcat-7/v7.0.61/bin/apache-tomcat-7.0.61.tar.gz"
+  sha1 "98ccbc95d029291e81bf818879ec8a0aa14465df"
```
Comment (DomT4) [inline tomcat7.rb]: Could you possibly change the ` sha1 `sums to ` sha256 `? Not mandatory, but would be appreciated.

Comment (asaph) [inline tomcat7.rb]: @DomT4 Done. I squashed it down to 1 commit.

Comment (DomT4) [inline tomcat7.rb]: Thanks! :smiley:

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew-versions/commit/569b61a14f4f325a4191d32c2d9786d543c2a0e8. Thanks for your contribution to Homebrew, we appreciate it! :smiley_cat:.

**Pull request closed without merging**